pkg/blobinfocache/internal/test/test.go

```diff
@@ -240,7 +248,7 @@ func testGenericCandidateLocations2(t *testing.T, cache blobinfocache.BlobInfoCa
 				{d: e.d, lr: e.n + "1"},
 				{d: e.d, lr: e.n + "2"},
 			}, cache.CandidateLocations(transport, scope, e.d, false))
-			assertCandidatesMatch2(t, scopeName, []candidate{}, cache.CandidateLocations2(transport, scope, e.d, false))
+			assertCandidatesMatch2UnknownLocation(t, scopeName, []candidate{}, cache.CandidateLocations2(transport, scope, e.d, false))
```

Comment:
These cases previously had the location data, why are they no longer providing that data?

Reply:
This change is not needed, my bad.